src/starcounter-debug-aid.html

```diff
@@ -173,6 +82,12 @@
                 //this.$.errorDialog.reset();
             },
             openDialog: function () {
+                if (!Polymer.isInstance(this.$.ui)) {
+                    var fileName = "starcounter-debug-aid.html";
+                    var link = document.querySelector('link[href$="' + fileName + '"]');
+                    var baseUrl = link.href.replace(fileName, '');
+                    Polymer.Base.importHref(baseUrl + "starcounter-debug-aid-ui.html");
```

Comment:
Why don't we use just `this.importHref`? and `this.resolveUrl`?
https://www.polymer-project.org/1.0/docs/devguide/instance-methods#imports-and-urls

or even `lazy-import` https://github.com/Polymer/lazy-imports ?
//cc @diliger @warpech